CONTRIBUTING.md

```diff
@@ -97,6 +97,17 @@ Changelog:
 - Does the pull request comply with the [conventional commits spec](https://www.conventionalcommits.org/en/v1.0.0/) and does the commit message convey the purpose of the change, see also [Compose a changelog relevant pull request](#compose-a-changelog-relevant-pull-request)?
 - Will this change highly impact the way we design APIs, and should this be discussed amongst API providers/API community?
 
+## Add a new guideline
+
+You are welcome to propose a new API guideline.
+To identify a rule ID, you can use the below command to retrieve a sorted list of the rule IDs already taken.
+This way you can identify a new rule ID for the guideline.
+Do not reuse existing rule IDs.
+
+```bash
+grep -r "^id:" api-guidelines | rev | cut -d" " -f1 | rev | sort | less
+```
```

Comment:
I'd like to provide a bit more context. What do you think @kiriakos ? :)

```suggestion
You are welcome to propose a new API guideline.
Each API guideline comes with a unique rule ID that is stated in the file front matter.
To identify an unused and thus available rule ID, run the following command:

```bash
grep -r "^id: R00" api-guidelines | rev | cut -d" " -f1 | rev | sort | less
```

You can choose any rule ID that is not included in the list.
```

Reply:
@cgebken,mentioning  the front matter is a very good point. I'd like to rephrase this a bit, WDYT?

> Each API guideline requires a unique rule ID, which must be noted front matter of the file.

Reply:
Updated my suggestions accordingly :)